contracts/core/BulkCreation.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {Metadata} from "./libraries/Metadata.sol";
+import {IRegistry} from "./interfaces/IRegistry.sol";
+
+contract BulkCreation {
+    IRegistry public constant REGISTRY = IRegistry(0xEB796bdb90fFA0f28255275e16936D25d3418603);
+
+    constructor() {}
+
+    function createProfiles(bytes memory encodedData) external {
+        (uint256[] memory _nonce, string[] memory _name, Metadata[] memory _metadata, address[] memory _owner) =
```

Comment:
add a project ID here? or add separate variable called projectIds. How to map the old project to the new profile on v2.